zap/src/main/java/org/zaproxy/zap/extension/pscan/scanner/StatsPassiveScanner.java

```diff
@@ -66,6 +71,19 @@ public void scanHttpResponseReceive(HttpMessage msg, int id, Source source) {
         }
     }
 
+    private static String getContentTypePostfix(String contentTypeValue) {
+        String[] ctvArray = contentTypeValue.split(";");
+        String contentType = ctvArray[0].trim();
+        if (ctvArray.length == 1) {
+            return contentType;
```

Comment:
Sorry for the conflicting comments, in the current version it could return without using the temporary variable.